### PR TITLE
updates readme for sdk react card

### PR DESCRIPTION
**Pull request by Nathan-Notifi**

### CODE REVIEW

File: packages/notifi-react-card/README.md [hunk 1]
```diff
@@ -11,6 +11,32 @@ npm install --save-dev @notifi-network/notifi-core
...
+    frameborder="0"
+    allow="autoplay; encrypted-media"
+    allowfullscreen
+>
+</iframe>
+
+### Design Guidelines
+
+We have general guidelines that we like to follow. You can see them here: [Figma](https://www.figma.com/file/ieF0Ynuc3WI608RCt7wKSf/Notifi-Template?node-id=0%3A1&t=v8zeo6UovJAOb9vR-0).
+
```
Comment (jennifer-notifi) [inline packages/notifi-react-card/README.md:31]: Suggestion: Should we show a final screenshot? Like an example of a partner site we really like? 

I have a feeling that people either won't look at the Figma or might have a hard time visualizing it without an example site or final screenshot of a live site.

**Review by jennifer-notifi — CHANGES_REQUESTED** [review]

File: packages/notifi-react-card/README.md [hunk 2]
```diff
@@ -11,6 +11,32 @@ npm install --save-dev @notifi-network/notifi-core
 
 You can import the default stylesheet to get baseline styling.
 
+## Video
+
+Below is a video useful for setting up the Notifi React Card.
+
+<iframe
```
[the remark this answers is not in the document]
Comment (Nathan-Notifi) [reply packages/notifi-react-card/README.md:18]: ah good catch

File: packages/notifi-react-card/README.md [hunk 3]
```diff
@@ -11,6 +11,32 @@ npm install --save-dev @notifi-network/notifi-core
 
 You can import the default stylesheet to get baseline styling.
 
+## Video
+
+Below is a video useful for setting up the Notifi React Card.
```
[the remark this answers is not in the document]
Comment (Nathan-Notifi) [reply packages/notifi-react-card/README.md:16]: got it

File: packages/notifi-react-card/README.md (hunk 1 above)
Comment (Nathan-Notifi) [reply packages/notifi-react-card/README.md:30]: done

**Review by jennifer-notifi — APPROVED** [review]
This looks good to me! Hopefully, we can spark a conversation about documentation as a whole with product. I would encourage you to have the conversation when you can :)

**Comment by Nathan-Notifi** [comment]
yeah, i think we scale, we'll def need to standardize some things

File: packages/notifi-react-card/README.md [hunk 4]
```diff
@@ -10,6 +10,33 @@ npm install --save-dev @notifi-network/notifi-core
...
+
+Here is a [link](https://www.youtube.com/watch?v=LIKu-clf9bg) covering how to setup the React Card Config.
+
+The video covers the following:
+
+- Configuring the card
+- Adding Event Types
+- Installing the react package into your project
+- Updating the default values to match your dapp.
+-
```
Comment (jennifer-notifi) [inline packages/notifi-react-card/README.md:25]: Hey Nathan, sorry just noticed this, can we please remove this bullet point?

**Pull request merged**